src/aura/BGE_DataImportBatchEntry/BGE_DataImportBatchEntryHelper.js

```diff
@@ -83,6 +83,22 @@
      * @description: invoke the Data Importer on all records associated with this Batch
      */
     processBatch: function(component) {
+        var totals = component.get('v.totals');
+        var record = component.get('v.record');
+        var labels = component.get('v.labels');
+        if (record[labels.requireTotalMatch]) {
+            if (record[labels.expectedCountField] == 0 && record[labels.expectedTotalField] == 0) {
```

Comment:
For 2 null values, this drops them into the second if. Should we actually throw this error message instead?

Reply:
I thought about this too. However I did not handle the "null values" use cases because these fields should not technically have null values. The Configuration Wizard does not save null values and the default values on the fields is 0.

Reply:
Fair enough! In that case, this is good for me, and I'll approve it.